jobs/e2e_node/crio/crio.ign

```diff
@@ -31,5 +36,13 @@
         "name": "crio-install.service"
       }
     ]
+  },
+  "passwd": {
+    "users": [
+      {
+        "name": "prow",
```

Comment:
If we change all of these configs, we have to remove the line that specifies the ssh user as core for all jobs, not just presubmits.

I'd suggest we instead copy this config, add a limited change, and change the jobs we want to test to use the new config.

Reply:
I don't think we do need to. when I did my limited testing, core user was still created, in addition to prow. However, it probably is a good idea just in case the new service messes something else up